web/packages/core/src/ruffle-player.ts

```diff
@@ -1197,7 +1197,7 @@ export class RufflePlayer extends HTMLElement {
         // TODO: Pause content until message is dismissed
         div.innerHTML = `<div class="message">
             <p>Flash Player has been removed from browsers in 2021.</p>
-            <p>This content is not yet supported by the Ruffle emulator and will likely not run as intended.</p>
+            <p>For more info check the #faq in <a href="discord.gg/ruffle>The Ruffle Discord </a></p>
```

Comment:
Please strip the trailing space before `</a>`.

Reply:
oops i'll fix it right now